src/garage/torch/distributions/tanh_normal.py

```diff
@@ -0,0 +1,154 @@
+"""A Gaussian distribution with tanh transformation."""
+from collections import namedtuple
+
+import torch
+from torch.distributions import Distribution, MultivariateNormal
+
+
+class TanhNormal(Distribution):
+    # pylint: disable=abstract-method
+    """A gaussian distribution that has a tanh transformation applied to it.
+
+    Algorithms like SAC and Pearl use this transformed distribution.
+    It can be thought of as a distribution of X where
+        Y ~ N(mean, cov)
```

Comment:
the `~` is usually meant to me "is distributed as" which is not what you mean here.

what you are saying is
```
.. :math:
   Y ~ \mathcal{N}(\mu, \sigma)
   X = tanh(Y)
```

also, this is an excellent opportunity to use the `:math:` environment in sphinx so that your docstring gets turned into rendered latex.